### DISCUSSION TITLE
Move KNOW THE PENALTIES TO AVOID THE PENALTIES to a more discreet yet useful location

### DISCUSSION THREAD

**Discussion by w0000000t** [bisq-network/bisq#6244] [Ideas]
As per conversation with @chimp1984 on matrix, this is a proposal to place the penalties warning in a less prominent place, saving on one additional popup, having the warning very close to where it is most needed, and also showing it each time the user is about to deal with trading, instead of only once at Bisq start.

So I propose to not show this anymore:
<img width="686" src="https://user-images.githubusercontent.com/95026987/173126460-b30bbdbf-6592-44ca-9a5d-e699317860cd.png">
and instead have this each time, right before user finalizes the making (or taking) of an offer:
![penalty](https://user-images.githubusercontent.com/95026987/173126578-9ed5d942-0a4e-44e0-9d2c-e5d52b943e54.jpg)

where I rendered the links as underlined blue text à-la-netscape

**Comment by MwithM** [comment]
Maybe this screen is already overloaded with more relevant info.
I like popups because  you can get rid of them, but  this is less invasive and concerning.

i wish there was a way to get rid of the word penalties while still inviting to read the rules.  Maybe _face the possibility of losing the security deposit_?

**Reply by chimp1984** [reply]
Yes to me penalties also comes over very negative. We dont want to push the user to the un-happy path which in fact happens only in about 5% of cases. I think the initial TAC screen is likely containing already the trade rules section. Users who take such terms serious will read it and others (like myself) will not read it anyway even if got stirred in the eye 100 times. I think most those rules are also common sense and if a user does something which is not totally straight (like using someone elses bank account) they can read up.

**Reply by w0000000t** [reply]
I get mixed signals here, on one side you both say it's less intrusive, on the other that it's still "too much info"... was that a polite way to say that the warning should be removed altogether?

**Reply by chimp1984** [reply]
I just checked the TACs:
`The user confirms that they have read and agreed to the rules regarding the dispute process`
I think there its enough IMO. 

Yes, I think the popup shoud be removed completely.

What should get added though would be a "Resources" tab in support with infos and links. @xyzmaker123 started once a menu item (which was too dominant as menu). I think that would be the right place to add such info.

**Reply by w0000000t** [reply]
Indeed I am sure there's plenty of hints in the app that point to the trading rules. I remember that when we decided to add the popup there had been a spike of disputes due to silly violations by users, that we thought a prominent heads-up was in order; in that occasion, I came up with the wording, to have the users  really check the rules and penalties, and not just "heh I'll read it sooner or later I guess, bye for now"
Might not be the case now anymore, and out mods can either confirm or deny.

**Reply by ghost** [reply]
Per the above discussion I've removed the penalties popup completely, within https://github.com/bisq-network/bisq/pull/6236
